shared/utils/testoval.py

```diff
@@ -4,8 +4,9 @@
 import os
 
 # always use shared/testoval_module.py version
-script_directory = os.path.dirname(__file__)
+script_directory = os.path.dirname(os.path.realpath(__file__))
 SHARED_MODULE_PATH = os.path.join(script_directory, "..", "modules")
+print script_directory
```

Comment:
yeah... no..

Reply:
@mpreisler lol. yeah totally didn't mean to commit that.

Reply:
@mpreisler just making sure that you were watching what I was doing. ;)
It should be fixed now.

Reply:
I see everything, Gah-bay! :D

Reply:
@mpreisler lol are you sauron the all seeing eye? :P